cpp/src/io/json/nested_json.hpp

```diff
@@ -110,6 +242,31 @@ void get_token_stream(device_span<SymbolT const> d_json_in,
                       SymbolOffsetT* d_tokens_indices,
                       SymbolOffsetT* d_num_written_tokens,
                       rmm::cuda_stream_view stream);
+
+/**
+ * @brief Parses the given JSON string and generates a tree representation of the given input.
+ *
+ * @param input The JSON input in host memory
+ * @param d_input The JSON input in device memory
+ * @param stream The CUDA stream to which kernels are dispatched
+ * @return
+ */
+json_column get_json_columns(host_span<SymbolT const> input,
+                             device_span<SymbolT const> d_input,
+                             rmm::cuda_stream_view stream);
+
+/**
+ * @brief Parses the given JSON string and generates a cudf::column of the given input.
+ *
+ * @param input The JSON input
+ * @param stream The CUDA stream to which kernels are dispatched
+ * @return cudf::column of the given input
+ */
+table_with_metadata parse_json_to_columns(
```

Comment:
Should this be `read_nested_json` or `parse_nested_json`, since it returns `table_with_metadata`?

Reply:
Renamed 👍